blocks/editable/index.js

```diff
@@ -287,8 +287,8 @@ export default class Editable extends Component {
 		}
 	}
 
-	onChange() {
-		if ( ! this.editor.isDirty() ) {
+	onChange( forced ) {
+		if ( ! forced && ! this.editor.isDirty() ) {
```

Comment:
Should we add a comment to explain that `isDirty` should be enough but due to a bug, we need to force sometimes?

Reply:
On talking to the TinyMCE team it's more of a known deficiency needed due to efficiency requirements.